src/main/java/com/cegeka/academy/web/rest/UserChallengeController.java

```diff
@@ -1,7 +1,24 @@
 package com.cegeka.academy.web.rest;
 
-import org.springframework.stereotype.Controller;
+import com.cegeka.academy.service.dto.UserChallengeDTO;
+import com.cegeka.academy.service.userChallenge.UserChallengeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
-@Controller
+import java.util.List;
+
+@RestController
+@RequestMapping("/userChallenge")
 public class UserChallengeController {
+
+    @Autowired
+    private UserChallengeService userChallengeService;
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET)
+    @ResponseBody
+    public List<UserChallengeDTO> getChallengesByUserId(@PathVariable("id") Long id){
```

Comment:
In service acest id este userId atunci si in controller ar trebui să l numești userId